lib/markdown_widget.dart

```diff
@@ -136,10 +140,13 @@ class _MarkdownWidgetState extends State<MarkdownWidget> {
   Widget buildMarkdownWidget() {
     return widget.controller == null
         ? ListView.builder(
+            shrinkWrap: widget.physics != null,
```

Comment:
Hi, I think `shrinkwrap` should not be associated with `scrollphysics`, you can add another property to set `shrinkwrap` 

Now `ScrollablePositionedList` dosen't support for  `shrinkwrap`([see here](https://github.com/google/flutter.widgets/issues/52)) , so this property is only valid for ListView (you need add these description for the `shrinkwrap` )

If you think it's OK and make the above changes, I will merge this pr quickly

(ps: you can also use `markdowngenerator ` to achieve the effect you want)